README.md

```diff
@@ -16,6 +16,35 @@ For details about configuration and deployment of RBD and CephFS CSI plugins, se
 
 For example usage of RBD and CephFS CSI plugins, see examples in `examples/`.
 
+## Support Matrix
+
+Here is the matrix that describes the features supported by the different
+versions of Ceph CSI driver
+
+|              |        Features             | CSI driver Version | Kubernetes Version   |
+|--------------|-----------------------------|--------------------|----------------------|
+| **CephFS**   | Snapshot                    |          -         |       >=v1.11        |
+|              | Clone from VolumeSnapshot   |          -         |       >=v1.13        |
+|              | Clone from VolumeSource     |          -         |       >=v1.13        |
+|              | Resize                      |          -         |         v1.14        |
+|              | RWX                         |   v0.3.0,v1.0.0    |       >=v1.11        |
+|              |                             |                    |                      |
+| **RBD**      | Snapshot                    |   v0.3.0,v1.0.0    |       >=v1.11        |
+|              | Clone from VolumeSnapshot   |       v1.0.0       |       >=v1.13        |
+|              | Clone from VolumeSource     |          -         |       >=v1.13        |
+|              | Resize                      |          -         |         v1.14        |
+|              | RWX(only block mode)        |       v1.0.0       |       >=v1.13        |
+|              | RWO                         |   v0.3.0,v1.0.0    |       >=v1.11        |
+
+The below matrix describes the compatibility of Ceph CSI driver with the
+CSI spec version
+
+| Ceph CSI driver Version | CSI spec version | Kubernetes version |
```

Comment:
We are very soon calling out v1.0.0 as alpha or even a release under the works, afaik. I would hence remove the line v1.0.0 from this table.